backend/src/fuesim-server.spec.ts

```diff
@@ -0,0 +1,51 @@
+import { jest } from '@jest/globals';
+import { Config } from 'config';
+import { uuid } from 'digital-fuesim-manv-shared';
+import { exerciseMap } from 'exercise/exercise-map';
+import { EntityManager } from 'typeorm';
+import { createExercise, createTestEnvironment } from '../test/utils';
+
+describe('Exercise saving', () => {
+    const environment = createTestEnvironment();
+    it('does not throw away actions while saving', async () => {
+        // Only run this test when a database gets used
+        expect(Config.useDb).toBe(true);
+
+        const exercideIds = await createExercise(environment);
+        const exercise = exerciseMap.get(exercideIds.trainerId)!;
+
+        const saveTick: () => Promise<void> = (environment.server as any)
+            .saveTick;
+
+        const savedEntities: unknown[] = [];
+
+        const saveMock = jest
+            .spyOn(EntityManager.prototype, 'save')
+            .mockImplementation((async (entities: unknown[]) => {
+                // Simulate adding an action while asynchronously waiting
+                exercise.applyAction(
+                    {
+                        type: '[AlarmGroup] Add AlarmGroup',
+                        alarmGroup: {
+                            alarmGroupVehicles: {},
+                            id: uuid(),
+                            name: 'Alarm Group',
+                        },
+                    },
+                    null
+                );
+                savedEntities.push(...entities);
+                return Promise.resolve(entities);
```

Comment:
```suggestion
                return entities;
```
The async function automatically converts this to a promise -> the Promise.resolve is unnecessary